mindsdb/integrations/handlers/lightwood_handler/setup.py

```diff
@@ -1,25 +1,32 @@
-from setuptools import setup, find_packages
+import setuptools
+import os
 
-pkgs, new_links = install_deps()
+about = {}
+with open("lightwood_handler/__about__.py") as fp:
+    exec(fp.read(), about)
 
-setup(
+with open('requirements.txt') as req_file:
+    requirements = [req.strip() for req in req_file.read().splitlines()]
+
+# TODO: relative import okay?
+with open('../../../../requirements_test.txt', 'r') as req_test_file:
```

Comment:
may be better will be hardcode that dependencies. Or avoid it, since it use only for tests